### DISCUSSION TITLE
[灌水] 怎么感觉遇到的自动贩卖机总是没找零...

### DISCUSSION THREAD

**Discussion by BKCake** [share121/inter-knot#3154] [其他]
![DSC04309](https://github.com/user-attachments/assets/7450cd03-68b6-4d52-8156-22866feeb1b0)
真的受不了，家在五分街，无论在家这边还是在光映广场，每次遇到贩卖机总是没有找零，每次都得凑齐丁尼才能买，不然就亏了...

**Comment by lllliufeng** [comment]
其实可以通过飞踢贩卖机来把多余的丁尼兑换成饮料（bushi

**Comment by icenow188** [comment]
确实，都有手机了怎么还不能手机支付
